### PR TITLE
Add integration description

### PR DESCRIPTION
**Pull request by sophieyfang**
b/229635463

### CODE REVIEW

File: integration_test/third_party_apps_data/applications/activemq/metadata.yaml [hunk 1]
```diff
@@ -1,6 +1,13 @@
 public_url: "https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/activemq"
 short_name: activemq
 long_name: Apache ActiveMQ
+description: -|
+  The Apache ActiveMQ integration collects storage usage and message metrics.
+  Storage metrics include memory and disk usage. Message metrics include number of
+  waiting messages, average wait time, and expired messages.
+
+  For more information about ActiveMQ, see
+  [activemq.apache.org/](https://activemq.apache.org/){:class="external"}.
```
Comment (qingling128) [inline integration_test/third_party_apps_data/applications/activemq/metadata.yaml:10]: Is the `{:class="external"}` part needed here, or can it be emitted or added later on internally.

Same for the other files.

**Review by qingling128 — APPROVED** [review]
LGTM modulo the open thread is addressed.

File: integration_test/third_party_apps_data/applications/activemq/metadata.yaml (hunk 1 above)
Comment (sophieyfang) [reply integration_test/third_party_apps_data/applications/activemq/metadata.yaml]: Removed the external as staging the doc with or without external tag there is no difference in the look

**Review by qingling128 — APPROVED** [review]
LGTM

**Pull request merged**